btcec/signature.go

```diff
@@ -37,43 +36,31 @@ var (
 	oneInitializer = []byte{0x01}
 )
 
-// Serialize returns the ECDSA signature in the more strict DER format.  Note
-// that the serialized bytes returned do not include the appended hash type
-// used in Bitcoin signature scripts.
-//
-// encoding/asn1 is broken so we hand roll this output:
-//
-// 0x30 <length> 0x02 <length r> r 0x02 <length s> s
+// Serialize returns the ECDSA signature as r || s,
+// where both r and s are encoded into 32 byte big endian integers.
+// s is in the smaller of the two signatures.
```

Comment:
"s is in the smaller of the two signatures": I know what you mean but the comment is a bit confusing. Briefly explain which two signatures you are referring to (`(r, -s)` and `(r, s)`). This is kinda explained in the related ADR but it would be nice if this was a bit more self-contained.